### PR TITLE
Feat migration to jina3

### PR DESCRIPTION
**Pull request by RuiyiYang01**

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -8,13 +11,23 @@ def check_query(resp):
             print(f'+- {m.uri}: {m.scores["cosine"].value:.6f}, {m.tags}')
 
 def main():
-    docs = DocumentArray(from_files('toy-data/*.mp3'))
-
+    cur_dir = os.path.dirname(os.path.abspath(__file__))
+    workspace = os.path.join(cur_dir, 'workspace')
```
Comment (nan-wang) [inline app.py:15]:
```suggestion
    from pathlib import Path
    workspace = Path(__file__).parent.absolute() / 'workspace'
```

File: app.py [hunk 2]
```diff
@@ -8,13 +11,23 @@ def check_query(resp):
...
+    logger = logging.getLogger('audio-search')
+    docs = DocumentArray.from_files('toy-data/*.mp3')
+    if os.path.exists(workspace):
```
Comment (nan-wang) [inline app.py:18]:
```suggestion
    if workspace.exists():
```

File: app.py [hunk 3]
```diff
@@ -1,5 +1,8 @@
-from jina import DocumentArray, Flow
-from jina.types.document.generators import from_files
+from jina import Flow
+from docarray import DocumentArray
+import os
```
Comment (nan-wang) [inline app.py:3]:
```suggestion
```

File: app.py [hunk 4]
```diff
@@ -1,5 +1,8 @@
...
+import sys
+import logging
```
Comment (nan-wang) [inline app.py:5]:
```suggestion
```

File: app.py [hunk 5]
```diff
@@ -8,13 +11,23 @@ def check_query(resp):
             print(f'+- {m.uri}: {m.scores["cosine"].value:.6f}, {m.tags}')
 
 def main():
-    docs = DocumentArray(from_files('toy-data/*.mp3'))
-
+    cur_dir = os.path.dirname(os.path.abspath(__file__))
+    workspace = os.path.join(cur_dir, 'workspace')
+    logger = logging.getLogger('audio-search')
```
Comment (nan-wang) [inline app.py:16]:
```suggestion
```

File: app.py [hunk 6]
```diff
@@ -8,13 +11,23 @@ def check_query(resp):
...
+    docs = DocumentArray.from_files('toy-data/*.mp3')
+    if os.path.exists(workspace):
+            logger.error(
```
Comment (nan-wang) [inline app.py:19]: let's just print because we are not logging to any file
```suggestion
            print(
```

File: requirements.txt [hunk 7]
```diff
@@ -1,2 +1,3 @@
-librosa>=0.8.0
-jina~=2.2
\ No newline at end of file
+librosa
+docarray
```
Comment (nan-wang) [inline requirements.txt:2]: fix the version for docarray as well
```suggestion
docarray
```

File: scripts/download_model.sh [hunk 8]
```diff
@@ -5,7 +5,7 @@ mkdir -p ${MODEL_DIR}
 if [ ! -f "${MODEL_DIR}/AudioCLIP-Full-Training.pt" ]; then
   echo "Downloading model"
   echo "------ Download AudioCLIP model ------"
-  wget -q https://github.com/AndreyGuzhov/AudioCLIP/releases/download/v0.1/AudioCLIP-Full-Training.pt
+  wget https://github.com/AndreyGuzhov/AudioCLIP/releases/download/v0.1/AudioCLIP-Full-Training.pt
```
Comment (nan-wang) [inline scripts/download_model.sh:8]: what's the motivation for this?

**Review by nan-wang — CHANGES_REQUESTED** [review]

File: scripts/download_model.sh (hunk 8 above)
Comment (Hippopotamus0308) [reply scripts/download_model.sh]: As the model is relatively large, user might doubt whether the downloading is stuck if they cannot see the process.(I downloaded the model for more than 5 minutes at home as the network is not so stable, so I think other users might meet up with similar problem if they cannot see the process)

**Review by nan-wang — APPROVED** [review]

**Pull request merged**